andela/balder-ah-backend
# #163283501 Modify favorite controller to return the correct res…

#### What does this PR do?
Modify the favorite endpoint to return correct responses needed for the frontend



#### Description of Task to be completed?
* Modify favorite controller
* Ensure all tests are passing



#### How should this be manually tested?
* Clone repo and `cd` into the directory
* Run `npm install` to install all the necessary dependencies
* Run `npm test` to ensure all tests are passing
* Run `npm run dev:start` to start the server and test the endpoint with postman



#### Any background context you want to provide?
N/A



#### What are the relevant pivotal tracker stories? (If applicable)
[#163283501](https://www.pivotaltracker.com/story/show/163283501)
